cves/CVE-2016-3082.yml

```diff
@@ -207,4 +225,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
A little more detail here would be nice :). Are you sure this is a coding mistake?

Reply:
Also, if you believe this is a coding mistake, how can coding mistakes be avoided in the future?

Reply:
You're right the wording here is lacking. I've rewritten this section to hopefully better reflect the situation.